src/PrestaShopBundle/Form/Admin/Sell/Product/Pricing/PricingType.php

```diff
@@ -95,6 +95,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'label_tag_name' => 'h2',
                 'label_help_box' => $this->trans('Set specific prices for customers meeting certain conditions.', 'Admin.Catalog.Help'),
             ])
+            ->add('catalog_price_rules', CatalogPriceRulesType::class, [
+                'label' => $this->trans('Catalog price rules', 'Admin.Catalog.Feature'),
+                'label_tag_name' => 'h2',
+                'label_help_box' => $this->trans('Set catalog price rules for customers meeting certain conditions.', 'Admin.Catalog.Help'),
```

Comment:
We don't use help boxes anymore on the new page, either a help message or nothing but I don't see a tooltip on the mockup